src/test_l1_dsAudio.c

```diff
@@ -4355,49 +5039,56 @@ void test_l1_dsAudio_negative_dsSetAudioGain(void) {
  * |06|Call dsAudioPortTerm() - Terminate audio ports | | dsERR_NONE |  Termination must be successful |
  * 
  */
-void test_l1_dsAudio_positive_dsGetAudioLevel(void) {
+void test_l1_dsAudio_positive_dsGetAudioLevel_sink(void) {
```

Comment:
where is the test case of source 
dsGetAudioLevel() 
https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsAudio.h#L1033